web_external/views/widgets/ScoreDetailWidget.js

```diff
@@ -34,6 +45,7 @@ var ScoreDetailWidget = View.extend({
             el.popover({
                 title,
                 container: this.$el,
+                viewport: '.c-submission-score-table',
```

Comment:
Can this be passed in as an element rather than a selector? If so, let's do that so it can be scoped to this widget.